src/setUpTests.js

```diff
@@ -0,0 +1,43 @@
+import { configure } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import configureStore from "redux-mock-store";
+import thunk from "redux-thunk";
+// To find out
+configure({ adapter: new Adapter() });
+
+function _bigMockStore_(initialState = {}, middlewares = []) {
+  // Global variables should have a _ at the beginning and at the end
+  const mockStore = configureStore([...middlewares, thunk]);
+  return mockStore(initialState);
+}
+
+// Takes your mounted wrapper, an array of components to look for, and whether you want to console log some details
+function _doesThisRender_(wrapper, components, logger = false) {
+  logger
```

Comment:
I added this function back cause it doubled our passing tests, can always be removed if necessary